ansible/lp_neo4j-backup.yml

```diff
@@ -1,4 +1,4 @@
-- hosts: learning-neo4j-node1  #if it is a cluster learning-neo4j-node1 should be always master node
+- hosts: db-backup-server  #server using for backups
```

Comment:
why can't we use the existing server host and not add another entry?

Reply:
@SMYALTAMASH The task update is to use a common db-backup server for all the db backup's.  
Is there is option to add the host as variables and say if the "db-backup-server" is defined then use the host values as that, if not then by default use "learning-neo4j-node1" as host.